samples/helloworld.c

```diff
@@ -0,0 +1,8 @@
+// Traditional "Hello World" program
+
+#include <stdio.h>
+
+int main(void) {
+   printf("Hello, World!\n");
+   return 0;
+}
```

Comment:
I follow @greg-king5's argument in https://github.com/cc65/cc65/pull/1500#pullrequestreview-678114487 - so please remove this file (and your change to samples/Makefile).

Reply:
I have removed helloworld.c from the samples directory and adjusted the Makefile accordingly.